PWG/EMCAL/EMCALbase/AliAnalysisTaskEmcal.h

```diff
@@ -324,7 +350,23 @@ class AliAnalysisTaskEmcal : public AliAnalysisTaskSE {
   void                        SetMinBiasTriggerClassName(const char *n)             { fMinBiasRefTrigger = n                              ; }
   void                        SetTriggerTypeSel(TriggerType t)                      { fTriggerTypeSel    = t                              ; } 
   void                        SetUseAliAnaUtils(Bool_t b, Bool_t bRejPilup = kTRUE) { fUseAliAnaUtils    = b ; fRejectPileup = bRejPilup  ; }
-  void                        SetUseInternalEventSelection(Bool_t doUse)            { fUseInternalEventSelection = doUse                  ; }
+
+  /**
+   * @brief Use internal (old) event selection
+   * @param[in] doUse It true use the old internal event selection instead of AliEventCuts
+   */
+  void                        SetUseInternalEventSelection(Bool_t doUse)            { fUseBuiltinEventSelection = doUse                  ; }
```

Comment:
I know that this name probably has to be maintained for backwards compatibility, but as you noted, we already used this term in the embedding helper for something rather different. Would you mind also adding a

```c++
void SetUseBuiltinEventSelection(Bool_t doUse)  { fUseBuiltinEventSelection = doUse; }
```

and marking the previous one as deprecated? I think this would help clarify the separate usage, especially since it's been renamed internally.